readme.md

```diff
@@ -82,6 +82,20 @@ Default: `false`
 
 Suppress all error messages. For example: `Process doesn't exist`.
 
+##### verify
+
+Type: `boolean`\
+Default: `true`
+
+Perform a rigorous check that the process is no longer visible to the system before returning.
```

Comment:
```suggestion
Perform a rigorous check that the process is no longer running before returning.
```

"visible" sounds weird. Makes me think it could be "hidden".